modules/olm-deleting-operators-from-a-cluster-using-cli.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-deleting-operator-from-a-cluster-using-cli-{context}']
+= Deleting operators from a cluster using the CLI
+
+Instead of using the {product-title} web console, you can delete an Operator
+from your cluster using the CLI.
+You do this by deleting the Subscription and ClusterServiceVersion
+from the targetNamespace, then editing the CatalogSourceConfig to remove
+the Operator's package name.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin`
+permissions.
+
+.Procedure
+
+In this example, there are two operators (Jaeger and Descheduler) installed in the
+openshift-operators namespace. The goal is to remove Jaeger without removing Descheduler.
+
+. Delete the Operator's subscription (for example, jaeger):
++
+----
+$ oc delete subscription jaeger -n openshift-operators
+subscription.operators.coreos.com "jaeger" deleted
+----
+
+. Check the current version of the subscribed Operator (for example, jaeger):
++
+----
+$ oc get subscription jaeger -n openshift-operators -o yaml | grep currentCSV
+  currentCSV: jaeger-operator.v1.8.2
+----
+
+. Delete the CSV for the Operator (jaeger) in the targetNamespace (openshift-operators)
++
+----
+$ oc delete clusterserviceversion jaeger-operator.v1.8.2 -n openshift-operators
+clusterserviceversion.operators.coreos.com "jaeger-operator.v1.8.2" deleted
+----
+
+.  List the CatalogSourceConfig to see the packages included in spec.packages.
++
+----
+$ oc get catalogsourceconfig -n openshift-marketplace \
+    installed-community-openshift-operators -o yaml
+----
+For example, the spec section might appear as follows:
++
+----
+spec:
+  csDisplayName: Community Operators
+  csPublisher: Community
+  packages: jaeger,descheduler
+  targetNamespace: openshift-operators
+----
+. Open the CatalogSourceConfig for editing as follows:
++
+----
+$ oc edit catalogsourceconfig -n openshift-marketplace \
+    installed-community-openshift-operators
+----
+
++
+Change the packages line to appear as follows:
++
+----
+  packages: descheduler
+----
+Save the change and the marketplace-operator will reconcile the CatalogSourceConfig.
+
+[NOTE]
+====
+If there is only one Operator in the CatalogSourceConfig, you can remove it
```

Comment:
Is this really a valid option? If so, I'd change the section around line 60 to offer it as alternative up front. Something like, 

>. Remove the Operator's package.
** If you have only one Operator...
** If you have multiple Operators..

Reply:
Great idea. I had to read about nested lists, but it's good now I think.